heatsynclabs/members_ui
# Refactor/function based components

Replaces #10

### ✏️ Changes

1. Creates event edit functionality
2. Totally changes around all the shit

### :wrench: Steps to test

1. Spin it all up and ensure fixtures are created
2. View an event
3. Go to its /edit URL
4. Make changes and hit save
